app/views/hosts/_form.html.erb

```diff
@@ -22,6 +23,10 @@
       <% end %>
       <li><a href="#params" id='params-tab' data-url='<%= current_parameters_hosts_path %>' data-url2='<%= puppetclass_parameters_hosts_path %>' data-toggle="tab"><%= _('Parameters') %></a></li>
       <li><a href="#info" data-toggle="tab"><%= _('Additional Information') %></a></li>
+
+      <% @tabs.keys.each do |tab| %>
```

Comment:
In my vision, host's aspect is any subject in the lifetime of a host that is not inventory - such as config, content e.t.c. Params and info are parts of the basic inventory - they should not belong to any aspect.